fs/procfs/fs_procfsmeminfo.c

```diff
@@ -578,6 +578,11 @@ static int meminfo_stat(FAR const char *relpath, FAR struct stat *buf)
 
 void procfs_register_meminfo(FAR struct procfs_meminfo_entry_s *entry)
 {
+  if (NULL == entry->name)
```

Comment:
should we add '__KERNEL__' too?

Reply:
Not necessary but we can add it to reduce code size.

Reply:
Done

Reply:
look like mm_initialize already check it:):
https://github.com/apache/incubator-nuttx/blob/master/mm/mm_heap/mm_initialize.c#L234-L236
we don't need check it inside procfs_register_meminfo again.

Reply:
@xiaoxiang781216 
OK, so we can remove `#ifdef CONFIG_KERNEL` which added to  `procfs_register_meminfo()`.

Reply:
Done